spec/imsc-hrm.html

```diff
@@ -195,7 +196,7 @@
 
     <p>Furthermore, the model is not intended to constrain readability complexity.</p>
   </section>
-  <section id='sotd'>
+  <section id='sotd' class='updateable-rec'>
     <p>For this specification to exit the CR stage, one of the following minimum
```

Comment:
I think we should remove these lines (lines 199-204 pre-pull request, 200-205 in the current commit) about CR exit from the PR version, or edit them.

Reply:
Implemented at https://github.com/w3c/imsc-hrm/pull/77/commits/436b4dfa4a23ded4c1bc7d04c65b52695e84f3d6

Reply:
Thank you, works for me. I have sent a short notice [message to TTWG](https://lists.w3.org/Archives/Public/public-tt/2024Feb/0005.html) highlighting this change and proposing to proceed with this by end of Thursday 22nd February if there are no objections. I think we're very much in a "last minute editorial tweaks" phase here, which I'd very much like to emerge from as soon as possible!

Reply:
No objections received. I'll confirm in a reply to that message; we can now continue with the CR exit criteria removed as per the current state of this pull request.